client-app/shared/catalog/components/image-gallery.vue

```diff
@@ -0,0 +1,192 @@
+<template>
+  <div :id="componentId" class="image-gallery">
+    <div class="image-gallery__images-container">
+      <template v-if="images?.length">
+        <div class="image-gallery__bg">
+          <VcIcon class="image-gallery__icon" name="search" size="xxl" />
+        </div>
+
+        <Swiper
+          class="image-gallery__images"
+          :modules="modules"
+          :thumbs="{ swiper: thumbsSwiper }"
+          :breakpoints="{
+            1280: {
```

Comment:
Let's use this constants here: https://github.com/VirtoCommerce/vc-theme-b2b-vue/blob/7f8b9f85ea2940a09c5350676d0fcb67021cc57c/client-app/core/constants/tailwind.ts#L6
May be with some util to extract value or refactor constants like
```typescript
const BREAKPOINT_VALUES: number[] = ...
const BREAKPOINTS = BREAKPOINT_VALUES.map()...
```